spec/models/case/sar/offender_spec.rb

```diff
@@ -76,6 +76,15 @@
     end
   end
 
+  describe "#external_deadline for rejected" do
+    let(:rejected_offender_sar_case) { create(:offender_sar_case, :rejected) }
+    let(:received_date) { Time.zone.today }
+
+    it "sets external deadline to 3 calendar months from received date" do
```

Comment:
It sets it to 90 days which won't always be 3 months, so the test description should reflect that.